index.html

```diff
@@ -15,10 +18,60 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+
+  <header class="txt-color-white">
+    <div class="row container logo">
```

Comment:
No one element should be both a `container` and a `row`. You should make the `row` a child of this element.